src/lio/bin/lio_mv.c

```diff
@@ -194,13 +197,15 @@ int main(int argc, char **argv)
         } else {
             info_printf(lio_ifd, 0, "ERROR: Multiple paths selected but the dest(%s) isn't a directory!\n", dtuple.path);
         }
+        n_errors = 1;
```

Comment:
Look at the line right after.  It goes and kicks out.  So +=1 or = 1 
accomplish the same thing.